src/routes/stalls.ts

```diff
@@ -23,4 +23,6 @@ router.post('/:id/reviews', auth, stallController.createStallReviewFuncs);
 router.post('/:id/favourites', auth, stallController.createStallFavouriteFuncs);
 router.delete('/:id/favourites', auth, stallController.destroyStallFavouriteFuncs);
 
+router.put('/:id/categoryStalls', auth, stallController.updateCategoryStallsFuncs);
```

Comment:
The url could probably be `PUT /:id/setCategoryStalls` to differentiate it from `POST /:id/reviews` because one is taking in ids as input and the other is taking the whole review.